<launch>
  <arg name="twist_dir_topic" default="/android_input_dir"/>
  <arg name="int_dir_topic" default="/pub_dir"/>
  <arg name="int_vel_topic" default="/pub_vel"/>
  <arg name="int_vel_max" default="108"/>
  <arg name="vel_cmd_topic" default="/android_input_vel"/>
  <arg name="bool_btn_topic" default="/android_input_velin"/>



    <node name="robot_driving" pkg="robot_driving" type="AndroidConversor.py" output="screen">
      <param name="twist_dir_topic" value="$(arg twist_dir_topic)"/>
      <param name="int_dir_topic" value="$(arg int_dir_topic)"/>
      <param name="int_vel_topic" value="$(arg int_vel_topic)"/>
      <param name="int_vel_max" value="$(arg int_vel_max)"/>
      <param name="vel_cmd_topic" value="$(arg vel_cmd_topic)"/>
    </node>

</launch>